bin/build-release-ccd-definition.sh

```diff
@@ -6,7 +6,7 @@ environment=${1:-prod}
 excludeNonProdFiles=${2:-true}
 
 if [ ${environment} == preview ]; then
-  excludedFilenamePatterns="-e *-nonprod.json"
+  excludedFilenamePatterns=""
```

Comment:
Just wondering if `nonprod.json` files were excluded from preview environments for a reason. @mounikahmcts do you know if this would impact QA negatively in any way?